frontend/src/components/elements/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,176 @@
+
+    const [activeKey, setActiveKey] = useState("0")
+    const tabListRef = useRef<HTMLUListElement>(null)
+
+    const isOverflowing = useIsOverflowing(tabListRef, false)
+
+    const theme = useTheme()
+
+    return (
+      <StyledTabContainer
+        isOverflowing={isOverflowing}
+        tabHeight={TAB_HEIGHT}
+        className="stTabs"
+      >
+        <UITabs
+          activeKey={activeKey}
+          onChange={({ activeKey }) => {
+            setActiveKey(activeKey.toString())
+          }}
+          disabled={widgetsDisabled}
+          overrides={{
+            TabHighlight: {
+              style: () => ({
+                backgroundColor: widgetsDisabled
+                  ? theme.colors.fadedText40
+                  : theme.colors.primary,
+                height: `2px`,
+                // Requires bottom offset to align with the TabBorder
+                bottom: `3px`,
+              }),
+            },
+            TabBorder: {
+              style: () => ({
+                backgroundColor: theme.colors.fadedText05,
+                height: `2px`,
+              }),
+            },
+            TabList: {
+              props: { ref: tabListRef },
+              style: () => ({
+                gap: theme.spacing.lg,
+                ...(isStale
+                  ? {
+                      opacity: 0.33,
+                      transition: "opacity 1s ease-in 0.5s",
+                    }
+                  : {}),
+              }),
+            },
+            Root: {
+              style: () => ({
+                // resetting transform to fix full screen wrapper
+                transform: `none`,
+              }),
+            },
+          }}
+          activateOnFocus
+        >
+          {node.children.map(
+            (appNode: AppNode, index: number): ReactElement => {
+              const childProps = {
+                ...props,
+                ...{ node: appNode as BlockNode },
+              }
+              let nodeLabel = index.toString()
+              if (childProps.node.deltaBlock?.tab?.label) {
+                nodeLabel = childProps.node.deltaBlock.tab.label
+              }
+              const isSelected = activeKey === index.toString()
+              const isLast = index === node.children.length - 1
+
+              return (
+                <UITab
+                  title={nodeLabel}
+                  key={index}
+                  overrides={{
+                    TabPanel: {
+                      style: () => ({
+                        paddingLeft: theme.spacing.none,
+                        paddingRight: theme.spacing.none,
+                        paddingBottom: theme.spacing.none,
+                        paddingTop: theme.spacing.lg,
+                      }),
+                    },
+                    Tab: {
+                      style: () => ({
+                        height: TAB_HEIGHT,
+                        whiteSpace: `nowrap`,
```

Comment:
nit: is there a reason this string is wrapped in backticks vs double quotes? (same comment in a few other places in this file)